akka-bench-jmh-dev/src/main/scala/akka/stream/MaterializationBenchmark.scala

```diff
@@ -20,7 +20,7 @@ object MaterializationBenchmark {
   }
 
   val graphWithJunctionsBuilder = (numOfJunctions: Int) =>
-    FlowGraph.closed() { implicit b =>
+    FlowGraph.runnable() { implicit b =>
```

Comment:
@rkuhn @drewhk @ktoso

If we want even more symmetry, we should add a `fromGraph` to RunnableGraph's companion object so that you can do:

~~~scala
val r = RunnableGraph.fromGraph( FlowGraph.create() { implicit b => ClosedShape() }
~~~

wdyt?